README.md

```diff
@@ -1,20 +1,222 @@
+
+    ----ENVIRONMENT-----------------
+
+    Runtime: Oracle Corporation OpenJDK 64-Bit Server VM 25.121-b13
+
+
+    ----ARGUMENTS-------------------
+
+    algorithm: PBEWithMD5AndDES
+    input: Ch4ll3ng3
+    password: db_password
+
+
+    ----OUTPUT----------------------
+
+    EHL/foiBKY2Ucy3oYmxdkFiXzWnOu7by
+
+The 'input' is your database password. The 'password' is a password specified by you that can be used to decrypt the data.
+The 'output' is your encrypted password. This encrypted password can then be used in the Circus Train yaml configuration:
+
+    housekeeping:
+      data-source:
+        #The package of your driver class
+        driver-class-name: com.mysql.cj.jdbc.Driver
+        #JDBC URL for your Database
+        url: jdbc:mysql://circus-train-housekeeping.foo1baz123.us-east-1.rds.amazonaws.com:3306/circus_train
+        #Database Username
+        username: bdp
+        #Encrypted Database Password
+        password: ENC(EHL/foiBKY2Ucy3oYmxdkFiXzWnOu7by)
+
+Or be decrypted from special properties file(s) on your classpath:
+
+        @Configuration
+	      @EncryptablePropertySources({@EncryptablePropertySource("classpath:encrypted.properties"), @EncryptablePropertySource("classpath:encrypted2.properties")})
+	      public class MyApplication {
+		         ...
+	      }
+
+The encrypted.properties file would look something like this:
+
+        database.username=ENC(nrmZtkF7T0kjG/VodDvBw93Ct8EgjCA+)
+        database.password=ENC(EHL/foiBKY2Ucy3oYmxdkFiXzWnOu7by)
+
+You can then access the decrypted username and password in your application by doing something akin to the following:
+
+        private @Autowired ConfigurableEnvironment env;
+
+        @Bean(destroyMethod = "close")
+        DataSource housekeepingDataSource(
+          String driverClassName,
+          String jdbcUrl) {
+
+          String username = env.getProperty("database.username");
+          String password = env.getProperty("database.password");
+
+          return DataSourceBuilder
+            .create()
+            .driverClassName(driverClassName)
+            .url(jdbcUrl)
+            .username(username)
+            .password(password)
+            .build();
+        }
+
+Or
+
+        @Bean(destroyMethod = "close")
+        DataSource housekeepingDataSource(
+          String driverClassName,
+          String jdbcUrl,
+          @Value("${database.username}") username,
+          @Value("${database.password}") password) {
+          return DataSourceBuilder
+            .create()
+            .driverClassName(driverClassName)
+            .url(jdbcUrl)
+            .username(username)
+            .password(password)
+            .build();
+        }
+
+
+Finally if you are using a encrypted password, when you run your application you must provide the application with your
+jasypt.encryptor.password there are a few approaches to doing so.
```

Comment:
" there are a few approaches to doing so." -> Make this a new sentence: "There are a few approaches to doing so:" with a : as you're about to list the approaches.